packages/graphql-yoga/__tests__/requests.spec.ts

```diff
@@ -402,6 +402,26 @@ describe('requests', () => {
     expect(body.errors?.[0].message).toBe('Unexpected parameter "test" in the request body.');
   });
 
+  it('does not error if there is a specified invalid parameter in the request body', async () => {
```

Comment:
Maybe also add a test of having a field that's not specified in the `extraParamNames`?

Reply:
Done!
https://github.com/dotansimha/graphql-yoga/pull/3333/commits/5f07980b9bb4861879f632317f458e16730113d2